.github/workflows/test-basics-install.yml

```diff
@@ -0,0 +1,31 @@
+# This is a basic workflow that is manually triggered
+
+name: Manual workflow
+
+# Controls when the action will run. Workflow runs when manually triggered using the UI
+# or API.
+on:
+  pull_request:
+    types: [ assigned, edited, opened, ready_for_review, reopened, synchronize ]
+    branches: [ main, master ]
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  basics_install:
+    # Run on a BigSur machine
+    runs-on: macos-latest
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # First clone the branch involved in the PR
+    - name: clone PR branch
+      uses: actions/checkout@master
+      with:
+        fetch-depth: 1
+        path: pull_request
+        ref: ${{ github.event.pull_request.head.ref }}
```

Comment:
I'm pretty sure this is not going to work with a manual run.  There is no notion of a PR (`pull_request`) when you do a manual run.

Reply:
True. Will adjust the trigger scope to PR only then as if we only allow PRs to update Master, we should be safe